### PR TITLE
Adding unit tests for cdn folder

### PR DESCRIPTION
**Pull request by srijeet0406**
This PR is not related to any issue. It increases the test coverage in the `cdn` folder.

<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Ops

## What is the best way to verify this PR?

Make sure the tests pass.
## If this is a bugfix, which Traffic Control versions contained the bug?

- master

## PR submission checklist
- [x] This PR has tests 
- [ ] This PR has documentation 
- [ ] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

File: traffic_ops/traffic_ops_golang/cdn/domains_test.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
+package cdn
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/jmoiron/sqlx"
+
+	"gopkg.in/DATA-DOG/go-sqlmock.v1"
+)
```
Comment (zrhoffman) [inline traffic_ops/traffic_ops_golang/cdn/domains_test.go:10]: Imports should come after the license

File: traffic_ops/traffic_ops_golang/cdn/named_delete_test.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
+package cdn
+
+import (
+	"testing"
+
+	"github.com/jmoiron/sqlx"
+
+	"gopkg.in/DATA-DOG/go-sqlmock.v1"
+)
```
Comment (zrhoffman) [inline traffic_ops/traffic_ops_golang/cdn/named_delete_test.go:9]: Another place where imports should come after the license

**Review by zrhoffman — CHANGES_REQUESTED** [review]
Just a small nit about license/import order

File: traffic_ops/traffic_ops_golang/cdn/queue_test.go [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+import (
+	"testing"
+
+	"github.com/jmoiron/sqlx"
+
+	"gopkg.in/DATA-DOG/go-sqlmock.v1"
+)
+
+/*
```
Comment (rimashah25) [inline traffic_ops/traffic_ops_golang/cdn/queue_test.go:11]: Another place where imports should come after the license

**Review by rimashah25 — CHANGES_REQUESTED** [review]
As soon as the license comments are addressed, the PR would LGTM. Tested all new unit tests locally and all seems to be working in order.

**Review by rimashah25 — APPROVED** [review]

**Review by zrhoffman — APPROVED** [review]
Looks good! The failing tests are not related to this PR.

**Pull request merged**